### PR TITLE
Add GitHub CLI

### PR DESCRIPTION
**Pull request by bartlomiejdanek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ itself) to look for plugins.
...
 | Fluxctl                       | [stefansedich/asdf-fluxctl](https://github.com/stefansedich/asdf-fluxctl)                         | [![Build Status](https://travis-ci.com/stefansedich/asdf-fluxctl.svg?branch=master)](https://travis-ci.com/stefansedich/asdf-fluxctl)                                                  |
 | Gauche                        | [sakuro/asdf-gauche](https://github.com/sakuro/asdf-gauche)                                       | [![Build Status](https://travis-ci.org/sakuro/asdf-gauche.svg?branch=master)](https://travis-ci.org/sakuro/asdf-gauche)                                                                |
+| GitHub CLI                    | [bartlomiejdanek/asdf-github-cli](https://github.com/bartlomiejdanek/github-cli)                  | [![Build Status](https://github.com/bartlomiejdanek/asdf-github-cli/workflows/CI/badge.svg?branch=master)](https://github.com/bartlomiejdanek/asdf-github-cli/actions)                 |
```
Comment (imbsky) [inline README.md]:
```diff
-[bartlomiejdanek/asdf-github-cli](https://github.com/bartlomiejdanek/github-cli)
+[bartlomiejdanek/asdf-github-cli](https://github.com/bartlomiejdanek/asdf-github-cli)
```

**Comment by imbsky** [comment]
Could you rebase this?

**Comment by imbsky** [comment]
The code looks good!

**Comment by pkoch** [comment]
Bump! I didn't know that this was a thing, so I made my own: https://github.com/pkoch/asdf-gh

Let's add this to the index so that less people reinvent the wheel. :/

**Comment by bartlomiejdanek** [comment]
@imbsky @pkoch thanks for the poke! I have missed previous notifications. Guess that it's ready to be merged.

**Comment by imbsky** [comment]
Thanks @bartlomiejdanek and @pkoch!

**Pull request merged**